src/client/src/apps/SpotlightRoute/Spotlight.tsx

```diff
@@ -64,6 +57,37 @@ const Contacting = styled.span`
 
 const INPUT_ID = 'spotlight'
 
+function getSuggestionsComponent(response: DetectIntentResponse, platform: PlatformAdapter, intent: string) {
```

Comment:
extracted this as it was gettign too big. notice that it's outside of the react component which I think is a good practice (leaving as less as possible inside the closure of react component)